src/server.ts

```diff
@@ -52,6 +52,13 @@ export async function createServer(
             clientId: `plugin-server-v${version}`,
             brokers: serverConfig.KAFKA_HOSTS.split(','),
             logLevel: logLevel.NOTHING,
+            ssl: process.env.KAFKA_CLIENT_CERT_B64
+                ? {
+                      cert: Buffer.from(process.env.KAFKA_CLIENT_CERT_B64!, 'base64'),
+                      key: Buffer.from(process.env.KAFKA_CLIENT_CERT_KEY_B64!, 'base64'),
+                      ca: Buffer.from(process.env.KAFKA_TRUSTED_CERT_B64!, 'base64'),
```

Comment:
lol I was going to say `atob` was going to blow my mind.
This isn't too bad either. I'm obviously a node expert :P